config/clients/js/template/tests/helpers/nocks.ts.mustache

```diff
@@ -129,9 +129,9 @@ export const getNocks = ((nock: typeof Nock) => ({
     return nock(basePath)
       .get(`/stores/${storeId}/changes`)
       .query({
-        type,
         page_size: pageSize,
-        continuation_token: contToken
+        continuation_token: contToken,
+        ...(type ? { type } : { })
```

Comment:
Do you need this?

Can't type be sent as undefined, like the other two fields?

Reply:
Unfortunately yes :/

If we pass `type` as `undefined` then nock will require `type` to be present in the query params (I think it converts it to an empty string), this gets around that by only adding the type parameter if it's defined. Thinking about it, I can pull out the object construction and do a if on type to make the intent more obvious if you'd prefer?

Reply:
No need, I think this is OK. It's understandable what it does.

It could trip me up in the future about the nock part, but the reason will become obvious once anyone tries to modify it.

Let's merge! 🚀